From e7b4ac9defc6fb9b41441c0e4d9218475bb0c380 Mon Sep 17 00:00:00 2001
From: ZaneZam <cyxman@yahoo.com>
Date: Sat, 16 Apr 2016 15:58:50 +0200
Subject: [PATCH 2/2] default.prop patch

---
 default.prop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/default.prop b/default.prop
index 9acff91..3897057 100644
--- a/default.prop
+++ b/default.prop
@@ -1,8 +1,8 @@
 #
 # ADDITIONAL_DEFAULT_PROPERTIES
 #
-ro.adb.secure=1
-ro.secure=1
+ro.adb.secure=0
+ro.secure=0
 ro.allow.mock.location=0
 ro.debuggable=1
 ro.zygote=zygote32
-- 
2.8.1

