From 7d467ffd64381f73c3755ec6ca2d2f1d53652d2a Mon Sep 17 00:00:00 2001
From: ZaneZam <cyxman@yahoo.com>
Date: Sat, 16 Apr 2016 23:17:37 +0200
Subject: [PATCH 1/2] init.rc patch

---
 init.rc | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/init.rc b/init.rc
index 78adacc..e5e64db 100755
--- a/init.rc
+++ b/init.rc
@@ -21,6 +21,9 @@ on early-init
     # Set the security context of /adb_keys if present.
     restorecon /adb_keys
 
+    # Boeffla kernel: set the security contexts of /sbin
+    restorecon_recursive /sbin
+
     mount debugfs /sys/kernel/debug /sys/kernel/debug mode=755
 
     start ueventd
@@ -776,3 +779,10 @@ service logcatd /system/bin/logcat -b all -v threadtime -v usec -v printable -D
     user logd
     group log
     writepid /dev/cpuset/system-background/tasks
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
2.8.1

