From 61ffa9595f24bbec7691a0eb84a07773ec18868e Mon Sep 17 00:00:00 2001
From: ZaneZam <cyxman@yahoo.com>
Date: Sun, 6 Jul 2014 16:42:58 +0200
Subject: [PATCH] init.rc patch

---
 init.rc | 18 +++++++++++-------
 1 file changed, 11 insertions(+), 7 deletions(-)

diff --git a/init.rc b/init.rc
index d419973..4ea4279 100755
--- a/init.rc
+++ b/init.rc
@@ -147,6 +147,10 @@ loglevel 3
 # This is needed by any process that uses socket tagging.
     chmod 0644 /dev/xt_qtaguid
 
+# ko files for exfat -> AP: modules instead compiled into kernel
+    insmod /lib/modules/exfat_core.ko
+    insmod /lib/modules/exfat_fs.ko
+
 on fs
 # mount mtd partitions
     # Mount /system rw first to give the filesystem a chance to save a checkpoint
@@ -732,13 +736,6 @@ service powersnd /system/bin/samsungpowersoundplay
      disabled
      oneshot
 
-service bootanim /system/bin/bootanimation
-    class main
-    user graphics
-    group graphics
-    disabled
-    oneshot
-
 service installd /system/bin/installd
     class main
     socket installd stream 600 system system
@@ -824,3 +821,10 @@ service drsd /system/bin/drsd
 
 on property:sys.boot_completed=1
     selinux_restore -e
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
1.8.4.5

